src/js/components/products/detail.tsx

```diff
@@ -51,12 +58,39 @@ const ProductDetail = ({ product, productSlug, doFetchObject }: Props) => {
     product.description &&
     (product.description.startsWith('<h1>') ||
       product.description.startsWith('<h2>'));
+
+  // some mock data for now
```

Comment:
The API is in place to get real data; you should be able to use `fetchObjects` with `objectType: 'project'` instead of this mock data.

Reply:
@duggiemitchell Also remember that you'll want to filter each projects query by product, e.g.:

```js
fetchObjects({
  objectType: 'project',
  filters: {
    product: product.id,
  },
  reset: true,
})
```